context/MqlContext/MqlContextProvider.tsx

```diff
@@ -210,7 +218,8 @@ function MqlContextProviderInternal({
       });
     }
   }, [externalConfig, mqlServerUrlData, token, mqlContext, mqlServerUrlOverride]);
-  console.log('clientVersion', clientVersion)
+
+
```

Comment:
extra whitespace 😉

Reply:
We need a linter/prettier on this repo.

Reply:
Backlog task! I'll file one when I'm back at my desk.